pkg/operator/resource/resourceapply/core.go

```diff
@@ -328,24 +329,37 @@ func ApplySecret(client coreclientv1.SecretsGetter, recorder events.Recorder, re
 }
 
 func SyncConfigMap(client coreclientv1.ConfigMapsGetter, recorder events.Recorder, sourceNamespace, sourceName, targetNamespace, targetName string, ownerRefs []metav1.OwnerReference) (*corev1.ConfigMap, bool, error) {
+	return SyncPartialConfigMap(client, recorder, sourceNamespace, sourceName, targetNamespace, targetName, nil, ownerRefs)
+}
+
+func SyncPartialConfigMap(client coreclientv1.ConfigMapsGetter, recorder events.Recorder, sourceNamespace, sourceName, targetNamespace, targetName string, syncedKeys sets.String, ownerRefs []metav1.OwnerReference) (*corev1.ConfigMap, bool, error) {
```

Comment:
add a `gdoc` please

Reply:
added godoc to all four of the sync functions :+1: